examples/tutorials/curation/plot_3_upload_a_model.py

```diff
@@ -0,0 +1,126 @@
+"""
+Upload a pipeline to Hugging Face Hub
+=====================================
+"""
+##############################################################################
+# In this tutorial we will upload a pipeline, trained in SpikeInterface, to the
+# `Hugging Face Hub <https://huggingface.co/>`_ (HFH). To do this, you need a HFH account.
+# If you do not want to make an account, you can simply share the model folder with colleagues.
+# There are also several ways to interaction with HFH: the way we propose here doesn't use
```

Comment:
I want to mention this up-front since, as a user, I put off by having to create an account to use a feature. So I want people to still keep interested, even if they're put off by the account thing. To try and isolate the information, I've subdivided the page in three subsections, hope this helps!